.github/issue_template.md

```diff
@@ -0,0 +1,22 @@
+Please consider the following before opening a ticket:
+
+- Do you have a general QUESTION? 
+  - Please use the mailing list instead: https://groups.google.com/group/akka-user/
+    As the issue tracker's goal is to track specifically actionable issues.
+
+- Do you want to real-time discuss USING AKKA?
+  - Please use the gitter chat: http://gitter.im/akka/dev
+    As it's a real-time chat, you may not get a response, in which case perhaps ask your question on akka-user
+  
+- Do you want to discuss DEVELOPING AKKA?
+  - Please use the "developer" gitter chat: http://gitter.im/akka/dev
+    This chat is aimed specifically at developing Akka, or discussing what to implement etc.
+    
+- If you suspect (or know) you found an ISSUE:
+  - If the ISSUE is about Akka-HTTP, please report it here: https://github.com/akka/akka-http
+  - If the ISSUE is about one of the Alpakka projects, please report it here: https://github.com/akka/alpakka
+  - Similarily about Akka Persistence plugins etc, please report in their respective repositories.
+  - Otherwise, please report the issue here in the main Akka issue tracker.
+  
+  
+  Thank you and happy hakking!
```

Comment:
shouldn't these things go into the yellow note when you click "New Issue" rather than into the issue template?
we currently have:
```
Before you open an issue please review the contributing and conduct guidelines for this repository.
```
but more can probably be added there. It must however be as short and precise as possible. People don't read, especially not wall of text.

Reply:
Yeah, seems the yellow box is completely ignored by people so thought this would be more visible hm...

Perhaps a 1 sentence version of this would be better?

Reply:
but the yellow box is not saying anything of this
it's rather annoying for us to have to remove that text each time we create an issue, and it will be annoying with all created issues that will include the text because people don't care or dare removing it

Reply:
Yeah those people too heh; The yellow box we can't influence more than it is now AFAIK.

Reply:
Yeah, sadly - would have been a good spot for links to akka-user etc